samples/graph-teams-tag/nodejs/client/src/components/create-tag.jsx

```diff
@@ -76,26 +80,62 @@ const CreateTag = props => {
 
     // Handler when user click on create button.
     const onCreateTagButtonClick = () => {
-        microsoftTeams.app.getContext().then(async (context) => {
-            if (tagName !== "" && tagDescription !== "") {
-                var membersToBeAdded = addSelfIfNotAdded(context.user.id);
-
-                var createTagDto = {
-                    id: "",
-                    displayName: tagName,
-                    description: tagDescription,
-                    membersToBeAdded: membersToBeAdded,
-                    membersToBeDeleted: []
-                };
-
-                var response = await axios.post(`api/teamtag?teamId=${context.team.groupId}`, createTagDto);
-                if (response.status === 201) {
-                    microsoftTeams.dialog.submit("Created successfully!");
-                }
-            }
+        var createTagDto;
+        microsoftTeams.app.initialize().then(() => {
+            microsoftTeams.app.getContext().then((context) => {
+                if (tagName !== "" && tagDescription !== "") {
+                            var membersToBeAdded = addSelfIfNotAdded(context.user.id);
+            
+                            createTagDto = {
```

Comment:
Rename this variable "createTagDto" as "createNewTag" at all the places.

Reply:
Changed it